packages/language-server/package.json

```diff
@@ -20,6 +19,7 @@
     "test": "tap --no-coverage test/**"
   },
   "dependencies": {
+    "@astrojs/svelte-language-integration": "*",
```

Comment:
Shouldn't this be a semver range?

Reply:
I wasn't really sure on how to proceed for an internal package like this, I hesitated with `latest` but I read somewhere that `latest` sometimes has issues in workspaces. We could just put the version of the package, for sure

Reply:
I think that's what we want, look at how we do it in the main `astro` package for example. Changesets will handle updating the version when it changes.